lib/utils/validation.ts

```diff
@@ -16,12 +16,13 @@ limitations under the License.
 
 import validEmail = require('@resin.io/valid-email');
 
-const APPNAME_REGEX = new RegExp(/^[a-zA-Z0-9_-]+$/);
+const APPNAME_REGEX = new RegExp(/^[a-za-z0-9_-]+$/);
```

Comment:
Either this is mistakenly lowercased or ... it's mistakenly lowercased? :)

Reply:
What on earth.... No idea how that happened.